src/main/java/core/basesyntax/strategy/StrategyImplementation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StrategyImplementation {
+    private static final Map<FruitTransaction.Operation, FruitShopOperationsHandler> processSelector
+            = new HashMap<>();
+
+    public StrategyImplementation() {
+        processSelector.put(FruitTransaction.Operation.SUPPLY, new SupplyHandler());
+        processSelector.put(FruitTransaction.Operation.BALANCE, new BalanceHandler());
+        processSelector.put(FruitTransaction.Operation.RETURN, new ReturnHandler());
+        processSelector.put(FruitTransaction.Operation.PURCHASE, new PurchaseHandler());
```

Comment:
create this map in the main and provide it to this class via the constructor

Reply:
done